core/src/main/resources/org/apache/spark/ui/static/utils.js

```diff
@@ -227,4 +227,12 @@ function createRESTEndPointForMiscellaneousProcess(appId) {
   }
   return uiRoot + "/api/v1/applications/" + appId + "/allmiscellaneousprocess";
 }
+
+function getBaseURI(){
+  let baseURI = document.baseURI;
+  if(!baseURI){
```

Comment:
```suggestion
  if (!baseURI){
```

Or, simply can we simply do `document.baseURI || document.URL`?